sqlmesh/engines/commands.py

```diff
@@ -49,8 +48,10 @@ class PromoteCommandPayload(PydanticModel):
 
 
 class DemoteCommandPayload(PydanticModel):
-    snapshots: t.List[SnapshotTableInfo]
+    target_snapshots: t.List[Snapshot]
```

Comment:
Why this changed? This entire module is a leftover from the Airflow integration and needs to be removed. Can you please create a follow-up PR doing that but revert this change in the meantime.

Reply:
Same goes for the `targets.py` module.